src/styles/index.scss

```diff
@@ -1,3 +1,300 @@
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: transform 300ms ease;
+  }
+
+  &__content {
+    width: $card-width;
+    height: $card-height;
+    border-radius: 5px;
+  }
+
+  &__image {
+    @include flex-size-center(160px, 134px);
+
+    background-image: url(../images/imac.jpeg);
+    margin-left: 19px;
+    margin-top: 32px;
+    background-size: cover; // або contain, в залежності від вашого бажання
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-left: $left-margin;
+    height: 36px;
+    width: 166px;
+
+    &-text {
+      @include text-style($text-size, 500, $text-color);
+
+      line-height: 18px;
+
+      &:hover {
+        color: #34568b;
+        transition: color 300ms ease;
+      }
+    }
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    margin-top: 4px;
+    margin-left: $left-margin;
+    line-height: 14px;
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+      color: $subtext-color;
+    }
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 16px;
+    margin-top: 16px;
+    flex-wrap: wrap;
+    width: 166px;
+
+    &-stars {
+      display: flex;
+    }
+
+    // Цикл для стилізації зірок (діти класу &__rating)
+    &-star {
+      background-image: url(../images/star.svg); // Зображення за замовчуванням для всіх зірок
+
+      // Цикл для стилізації активних зірок
+      @for $i from 1 through $stars-count {
+        &:nth-child(#{$i}) {
+          @if $i <= $active-stars {
+            background-image: url(../images/star-active.svg); // Заповнена зірка для активних
+          }
+
+          @include star-position; // Позиціонування зірки
+        }
+      }
+      &:first-child {
+        margin-left: 0; // або просто видалити марджин
+      }
+    }
+
+    &-text {
+      font-size: 10px;
+      line-height: 14px;
+      text-align: right;
+
+      color: $text-color;
+    }
+  }
+
+  &__price {
+    @include flex-size-center(166px, 16px);
+
+    margin-top: 24px;
+    margin-left: $left-margin;
+
+    &-text {
+      @include text-style($text-size, 400, $subtext-color);
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      margin-left: auto;
+      color: $text-color;
+    }
+  }
+
+  &__buy-wrapper {
+    @include flex-size-center(
+      166px,
+      40px
+    ); // Центрування та розмір для обгортки
+
+    margin-top: 16px;
+    margin-left: $left-margin;
+    background-color: $button-color;
+    border: 1px solid $button-color;
+    border-radius: $border-radius;
+
+    a.card__buy {
+      @include text-style(
+        $buy-button-font-size,
+        $buy-button-font-weight,
+        white
+      );
+
+      text-transform: uppercase;
+      text-decoration: none;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: 100%;
+      height: 100%;
+    }
```

Comment:
It's better to avoid deep nesting in your SCSS. Deep nesting makes the code harder to read and understand. It also increases the specificity of your selectors which can lead to difficulties in overriding styles later. Try to limit your nesting to 1 or 2 levels deep.